### PR TITLE
javascript-forms completed

### PR DESCRIPTION
**Pull request by rachaelkim831**
form events

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+$input.addEventListener('blur', function handleBlur(evet) {
+  console.log('blur event was fired');
+  console.log('event.target.name', event.target.name);
+});
+
+$input.addEventListener('input', function handleInput(event) {
+  console.log('value of', event.target.name, event.target.value);
+});
+// if ($spans[index].textContent === eventkey)
```
Comment (scbowler) [inline javascript-forms/form-events.js:15]: All 3 events need to be added to all 3 inputs, `name`, `email`, and `message`

File: javascript-forms/form-values.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var $submit = document.getElementById('contact-form');
+function logSubmit(event) {
+  event.preventDefault();
+  var $name = $submit.elements[0];
+  var $email = $submit.elements[1];
+  var $text = $submit.elements[2];
```
Comment (scbowler) [inline javascript-forms/form-values.js:6]: While this works it is a bit risky and unclear which element is which. A better approach would be to use the inputs name attribute to get the value: `$submit.elements.email.value`

**Review by scbowler — CHANGES_REQUESTED** [review]
Moving in the right direction but a few updates needed. Let me know if you have any questions.

**Comment by rachaelkim831** [comment]
form events

File: javascript-forms/form-values.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  event.preventDefault();
+  var $name = $submit.elements.name.value;
+  var $email = $submit.elements.name.email;
+  var $text = $submit.elements.name.text;
+  var $newobject = {
+    name: $name,
+    email: $email,
+    message: $text
+  };
+  console.log($newobject);
```
Comment (scbowler) [inline javascript-forms/form-values.js:12]: Does this work? I think it should be:

```JAVASCRIPT
  var $name = $submit.elements.name.value;
  var $email = $submit.elements.email.value;
  var $text = $submit.elements.message.value;
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking better! One more update needed for `form-values.js`, make sure to test your code before submitting a pull request. Let me know if you have any questions.

**Comment by rachaelkim831** [comment]
form values

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**